dev-libs/ntl/ntl-11.4.4.ebuild

```diff
@@ -26,6 +26,11 @@ REQUIRED_USE="bindist? ( !cpu_flags_x86_avx2 )"
 
 DOCS=( "${WORKDIR}/${P}"/README )
```

Comment:
This is probably unnecessary because `einstalldocs` automatically installs README

Reply:
Aaah no it isn't, never mind